tests/unit/channel_test.py

```diff
@@ -26,6 +26,12 @@ def ssh_channel(ssh_client_session):
         chan.close()
 
 
+@pytest.mark.xfail(
+    reason='This test causes SEGFAULT, flakily. '
+    'Ref: https://github.com/ansible/pylibssh/issues/57',  # noqa: WPS326
+    run=False,
```

Comment:
Let's drop this so that it always runs at least
```suggestion
```

Reply:
fixed